crowbar_framework/app/models/service_object.rb

```diff
@@ -1461,6 +1461,12 @@ def add_role_to_instance_and_node(barclamp, instance, name, prop, role, newrole)
     true
   end
 
+  def add_role_to_proposal_instance_and_node(barclamp, instance, name, newrole)
+      role = RoleObject.find_role_by_name "#{instance}-config-#{inst}"
```

Comment:
Style/IndentationWidth: Use 2 (not 4) spaces for indentation. (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)